Adapters/Vungle/ISVungleAdapter/ISVungleBannerDelegate.m

```diff
@@ -11,29 +11,23 @@
 @implementation ISVungleBannerDelegate
 
 - (instancetype)initWithPlacementId:(NSString *)placementId
-                      containerView:(UIView *)containerView
                         andDelegate:(id<ISBannerAdapterDelegate>)delegate {
     self = [super init];
     if (self) {
         _placementId = placementId;
-        _containerView = containerView;
         _delegate = delegate;
     }
     return self;
 }
 
-#pragma mark - Banner Delegate
+#pragma mark - VungleBannerView Delegate
 
-- (void)bannerAdDidLoad:(VungleBanner * _Nonnull)banner {
+- (void)bannerAdDidLoad:(VungleBannerView * _Nonnull)bannerView {
     LogAdapterDelegate_Internal(@"placementId = %@", self.placementId);
-
-    [self.delegate adapterBannerDidLoad:self.containerView];
-    dispatch_async(dispatch_get_main_queue(), ^{
-        [banner presentOn:self.containerView];
-    });
+    [self.delegate adapterBannerDidLoad:bannerView];
 }
 
-- (void)bannerAdDidFailToLoad:(VungleBanner * _Nonnull)banner
+- (void)bannerAdDidFailToLoad:(VungleBannerView * _Nonnull)bannerView
```

Comment:
@ashinagawa please update the method. 
we noLonger have bannerAdDidFailToLoad:
we only have bannerAdDidFail:
`- (void)bannerAdDidFail:(VungleBannerView *)bannerView withError:(NSError *)withError`

Also looks like IS only needs load errors. 
please update the implementation accordingly.

Reply:
@ManojBudumuru-vungle 
I checked the old code (with `VungleBanner`), and we never sent `adapterBannerDidFailToShowWithError`.
Also, since we only have `bannerAdDidFail` for `VungleBannerView`, adapter can not know if it's failed in ad load, or ad present.

Reply:
As I discussed with @ManojBudumuru-vungle, I updated code to send `adapterBannerDidFailToShowWithError` or `adapterBannerDidFailToLoadWithError` based on the ad load status.